### DISCUSSION TITLE
Is it possible to filter http traffic by the X-Forwarded-For header?

### DISCUSSION THREAD

**Discussion by richterdennis** [friendly-bits/geoip-shell#9] [Q&A]
I want to filter incoming http traffic from a proxy. The original request ip is in the X-Forwarded-For header. Is it possible?

**Comment by friendly-bits** [comment]
Hi, geoip-shell does not support this functionality, and I am not aware of any way to implement it. I think you need to run geoip-shell on the proxy server.

**Reply by kiwilad-nz** [reply]
Just used your script. Looks like it's installed fine without issues. How do I go about testing this though as I've only allowed NZ as far as I am aware.

**Reply by friendly-bits** [reply]
Hi, please open a separate issue or discussion for this question and I'll gladly help you.
